src/Telemetry/ExcludedException.cs

```diff
@@ -20,7 +20,7 @@ public class ExcludedException : Exception
         public Type ExcludedType => InnerException.GetType();
 
         public ExcludedException(Exception innerException)
-            : base (innerException.Message, innerException)
+            : base (innerException?.Message, innerException)
```

Comment:
ExcludedException requires that innerException be non-null. If we use null propagation here, we should add an explicit check in the ctor:
```
           if (innerException == null) throw new ArgumentNullException(nameof(innerException));
```

Reply:
Fixed